Automattic/wp-calypso
# Media: No Content - Migrate Image to Webpack

#### Changes proposed in this Pull Request

This migrates the one image in `/media` to Webpack. For details on the reasoning behind this, see #36850.

#### Testing instructions

There should only be one use of this image: https://github.com/Automattic/wp-calypso/search?q=illustration-media.svg&unscoped_q=illustration-media.svg

You can find it by visiting an empty section of the Media Library. Verify that the image is still there and no different. 

<img width="572" alt="Screenshot 2019-10-19 at 16 30 40" src="https://user-images.githubusercontent.com/43215253/67147546-cde29580-f28d-11e9-81be-134947123270.png">
